src/sentry/api/serializers/models/integration.py

```diff
@@ -22,16 +23,53 @@ def serialize(self, obj, attrs, user):
         }
 
 
-class IntegrationIssueSerializer(IntegrationSerializer):
+class IntegrationIssueConfigSerializer(IntegrationSerializer):
     def __init__(self, group, action):
         self.group = group
         self.action = action
 
     def serialize(self, obj, attrs, user):
-        data = super(IntegrationIssueSerializer, self).serialize(obj, attrs, user)
+        data = super(IntegrationIssueConfigSerializer, self).serialize(obj, attrs, user)
         installation = obj.get_installation()
 
         if self.action == 'link':
             data['linkIssueConfig'] = installation.get_link_issue_config(self.group)
 
         return data
+
+
+class IntegrationIssueSerializer(IntegrationSerializer):
+    def __init__(self, group):
+        self.group = group
+
+    def get_attrs(self, item_list, user, *args, **kwargs):
+        external_issues = ExternalIssue.objects.filter(
```

Comment:
Can you filter this on integration id as well?

```py
external_issues = ExternalIssue.objects.filter(
    id__in=...,
    integration_id__in=[i.id for i in item_list],
)
```

Reply:
oh yeah good call